conf/airframes/TUDELFT/tudelft_origami_lisamxs_wifi_indi_stereoboard.xml

```diff
@@ -0,0 +1,320 @@
+    <define name="REF_MAX_RDOT" value="RadOfDeg(900.)"/>
+
+    <!-- feedback -->
+    <define name="PHI_PGAIN" value="400"/>
+    <define name="PHI_DGAIN" value="200"/>
+    <define name="PHI_IGAIN" value="100"/>
+
+    <define name="THETA_PGAIN" value="400"/>
+    <define name="THETA_DGAIN" value="200"/>
+    <define name="THETA_IGAIN" value="100"/>
+
+    <define name="PSI_PGAIN" value="500"/>
+    <define name="PSI_DGAIN" value="300"/>
+    <define name="PSI_IGAIN" value="10"/>
+
+    <!-- feedforward -->
+    <define name="PHI_DDGAIN" value="0"/>
+    <define name="THETA_DDGAIN" value="0"/>
+    <define name="PSI_DDGAIN" value=" 300"/>
+
+    <define name="PHI_AGAIN" value="0"/>
+    <define name="THETA_AGAIN" value="0"/>
+    <define name="PSI_AGAIN" value="0"/>
+  </section>
+
+  <section name="INS" prefix="INS_">
+     <!--define name="INT_GPS_ID" value="ABI_DISABLE"/-->
+     <define name="USE_GPS_ALT" value="true"/>
+     <define name="SONAR_UPDATE_ON_AGL" value="true"/>
+  </section>
+
+ <section name="GUIDANCE_V" prefix="GUIDANCE_V_">
+<!-- Guidance in vertical plane especially for use of range sensor-->
+   <define name="HOVER_KP" value="300"/>
+   <define name="HOVER_KD" value="55"/>
+   <define name="HOVER_KI" value="32"/>
+   <define name="NOMINAL_HOVER_THROTTLE" value="0.60"/>
+   <define name="ADAPT_THROTTLE_ENABLED" value="false"/>
+  </section>
+
+  <section name="AHRS" prefix="AHRS_">
+    <define name="H_X" value=" 0.47577"/>
+    <define name="H_Y" value=" 0.11811"/>
+    <define name="H_Z" value=" 0.87161"/>
+
+    <define name="GRAVITY_HEURISTIC_FACTOR" value="0"/>
+  </section>
+
+ <section name="GUIDANCE_H" prefix="GUIDANCE_H_">
+   <define name="PGAIN" value="0"/><!--80-->
+   <define name="DGAIN" value="260"/><!--260-->
+   <define name="IGAIN" value="0"/>
+   <define name="MAX_BANK" value="RadOfDeg(15)"/>
+   <define name="REF_MAX_SPEED" value="0.5"/>
+   <define name="USE_SPEED_REF" value="TRUE"/>
+
+ </section>
+
+  <section name="MISC">
+    <define name="VoltageOfAdc(adc)" value="(adc)*0.00162f" />
+  </section>
+
+ <section name="SIMULATOR" prefix="NPS_">
+   <define name="ACTUATOR_NAMES" value="ne_motor, se_motor, sw_motor, nw_motor" type="string[]"/>
+   <define name="JSBSIM_MODEL" value="simple_x_quad_ccw" type="string"/>
+   <define name="SENSORS_PARAMS" value="nps_sensors_params_default.h" type="string"/>
+ </section>
+
+ <modules main_freq="512">
+   <!--module name="gps" type="ubx_ucenter"/-->
+   <module name="send_imu_mag_current"/>
+<!--Stereo camera modules: Reintergrate this with the newest pprzlink!!>
+   <module name="stereocam">
+     <define name= "STEREO_BODY_TO_STEREO_PHI" value="90" unit="deg"/>
+     <define name= "STEREO_BODY_TO_STEREO_THETA" value="0" unit="deg"/>
+     <define name= "STEREO_BODY_TO_STEREO_PSI" value="90" unit="deg"/>
+     <define name="STEREO_UART" value="UART1"/>
+   </module-->
+   <module name="mag_pitot_uart.xml">
```

Comment:
You shouldn't use the mag_pitot_uart module to read the messages but add a module specially to do that.